src/afl-forkserver.c

```diff
@@ -274,6 +277,9 @@ void afl_fsrv_init_dup(afl_forkserver_t *fsrv_to, afl_forkserver_t *from) {
   fsrv_to->child_kill_signal = from->child_kill_signal;
   fsrv_to->fsrv_kill_signal = from->fsrv_kill_signal;
   fsrv_to->debug = from->debug;
+  fsrv_to->custom_mutator_data = from->custom_mutator_data;
+  fsrv_to->custom_fuzz_run = from->custom_fuzz_run;
+  fsrv_to->shm_id = from->shm_id;
```

Comment:
btw this is not really correct, it cannot have the same shm_id, so better to set this to -1, sorry.